api/types.go

```diff
@@ -293,7 +293,6 @@ type ProcessResponse struct {
 // ListModelResponse is a single model description in [ListResponse].
 type ListModelResponse struct {
 	Name       string       `json:"name"`
-	Model      string       `json:"model"`
```

Comment:
We deprecated `Name` in other places, so we should keep `Model` in the API types here and deprecate `Name`